opensrp-giz-malawi/src/main/java/org/smartregister/giz/activity/ChildDetailTabbedActivity.java

```diff
@@ -248,5 +248,10 @@ protected String getReportDeceasedMetadata() {
         }
         return "";
     }
+
+    @Override
+    public void updateViews() {
+        // Do nothing
+    }
```

Comment:
Why is this being overriden?

Reply:
It's abstract and needs to be implemented